app/models/user.rb

```diff
@@ -30,7 +30,7 @@ def platform_admin?
   end
 
   def charity_admin?
-    roles.exists?(name: "charity_admin")
+    roles.exists?(name: "charity_admin") || roles.exists?(name: "charity_original_admin")
```

Comment:
We're now using primary_charity_admin instead of charity_original_admin. But do we want both statuses covered in this method?